ghost/i18n/locales/pt-BR/portal.json

```diff
@@ -97,10 +97,10 @@
     "In the event a permanent failure is received when attempting to send a newsletter, emails will be disabled on the account.": "No caso de uma falha permanente ser recebida ao tentar enviar uma newsletter, os e-mails serão desativados na conta.",
     "In your email client add {{senderEmail}} to your contacts list. This signals to your mail provider that emails sent from this address should be trusted.": "No seu cliente de e-mail, adicione {{senderEmail}} à sua lista de contatos. Isso sinaliza ao seu provedor de e-mail que os e-mails enviados deste endereço devem ser confiáveis.",
     "Invalid email address": "Endereço de e-mail inválido",
-    "Jamie Larson": "Fulano(a) de tal",
-    "jamie@example.com": "fulano@exemplo.com",
+    "Jamie Larson": "Seu Nome",
```

Comment:
Some members of the Ghost dev team have strong feelings that this should be a recognizable generic name in your language.  Fulano/a de tal seems to accomplish that better than seu nome.

Reply:
I am ok with this

Reply:
@zenditimo , I think this is the only thing left, if you have time to have a look.

Reply:
Unfortunately, there is no person's name, such as "Jamie Larson", in Brazilian that can be used as a generic name to refer to as an example.

The word "Fulano" can come across as dismissive or derogatory, we Brazilians typically use when referring to someone in an indifferent or uninterested way. It often means a person of little importance, usually seen as lacking any notable qualities.

Adding "de tal" to "Fulano" makes this tone even stronger, highlighting the person's anonymity or lack of relevance. While "Fulano" alone already suggests indifference, "de tal" adds a sense of disregard or lack of effort to acknowledge or appreciate the person.

"Nome" or "Seu nome" what we actually use.

Below are two examples, the first from a Brazilian government website and the second is an Amazon form in Portuguese.

https://servicos.receita.fazenda.gov.br/servicos/cpf/inscricaopublica/inscricao.asp

https://www.amazon.com.br/ap/register?openid.pape.max_auth_age=0&openid.return_to=https%3A%2F%2Fwww.amazon.com.br%2F%3Fref_%3Dnav_ya_signin&prevRID=BKX6371DVR2CAYDT99H9&openid.identity=http%3A%2F%2Fspecs.openid.net%2Fauth%2F2.0%2Fidentifier_select&openid.assoc_handle=brflex&openid.mode=checkid_setup&prepopulatedLoginId=&failedSignInCount=0&openid.claimed_id=http%3A%2F%2Fspecs.openid.net%2Fauth%2F2.0%2Fidentifier_select&pageId=brflex&openid.ns=http%3A%2F%2Fspecs.openid.net%2Fauth%2F2.0

Reply:
We definitely don't want to be derogatory, so thanks for flagging this!

I didn't see a placeholder in the government form, but yes, I see what you mean about Amazon's use:
![image](https://github.com/user-attachments/assets/4f1bac26-9f82-4c88-aef7-d8d7f54be885)